artiq/frontend/artiq_coremgmt.py

```diff
@@ -132,6 +143,21 @@ def main():
             for key, filename in args.file:
                 with open(filename, "rb") as fi:
                     mgmt.config_write(key, fi.read())
+        if args.action == "write_ch_names":
+            ddb = DeviceDB(args.device_db).get_device_db()
+            channel_ntn = channel_number_to_name(ddb)
+            if not channel_ntn:
+                print("No device with channel number is found in device_db.py")
```

Comment:
I already told you several times that it's not called ``device_db.py``. Are you paying attention?